WordPress/src/main/res/values/strings.xml

```diff
@@ -2974,10 +2974,10 @@
     <string name="permission_microphone">Microphone</string>
 
     <!-- Image Optimization promo -->
-    <string name="turn_on">Turn on</string>
-    <string name="leave_off">Leave off</string>
-    <string name="image_optimization_promo_title">Turn on image optimization?</string>
-    <string name="image_optimization_promo_desc">Image optimization shrinks images for quicker uploading.\n\nYou can change this any time in site settings.</string>
+    <string name="turn_on">Yes, leave on</string>
+    <string name="leave_off">No, turn off</string>
+    <string name="image_optimization_promo_title">Keep optimizing images?</string>
+    <string name="image_optimization_promo_desc">Image optimization shrinks images for faster uploading.\n\This option is enabled by default, but you can change it in the app settings at any time.</string>
```

Comment:
I noticed one of the break lines (`\n`) only contains `\` after the change:

```suggestion
    <string name="image_optimization_promo_desc">Image optimization shrinks images for faster uploading.\n\nThis option is enabled by default, but you can change it in the app settings at any time.</string>
```

Reply:
Committed that change, thanks for spotting!